apps/E2E/src/Menu/specs/Menu.spec.win.ts

```diff
@@ -71,4 +73,110 @@ describe('Menu Functional Testing', () => {
+    await expect(await MenuPageObject.didItemCallbackFire()).toBeTruthy();
+    await expect(await MenuPageObject.didAssertPopup()).toBeFalsy(MenuPageObject.ERRORMESSAGE_ASSERT);
+  });
+
+  it('Validate MenuItem onClick() callback fired -> Press Space', async () => {
+    // open menu
+    await MenuPageObject.clickComponent();
+    await MenuPageObject.waitForMenuToOpen();
+    // fire callback
+    await MenuPageObject.sendKey(MenuPageObject.getItemSelector(1), Keys.SPACE);
+    await MenuPageObject.waitForItemCallbackToFire();
+    await expect(await MenuPageObject.didItemCallbackFire()).toBeTruthy();
+    await expect(await MenuPageObject.didAssertPopup()).toBeFalsy(MenuPageObject.ERRORMESSAGE_ASSERT);
+  });
+
+  it('Validate disabled MenuItem onClick() callback does not fire -> Click', async () => {
+    // we want to test for this because a disabled item is focusable for accessibility reasons, but it should not do anything when activated
+    await MenuPageObject.clickComponent();
+    await MenuPageObject.waitForMenuToOpen();
+    // fire callback
+    await MenuPageObject.clickComponent(MenuPageObject.getItemSelector(2));
+    await expect(await MenuPageObject.didItemCallbackFire()).toBeFalsy();
+
+    await expect(await MenuPageObject.didAssertPopup()).toBeFalsy(MenuPageObject.ERRORMESSAGE_ASSERT);
+  });
+
+  it('Validate disabled MenuItem onClick() callback does not fire -> Press Enter', async () => {
+    // we want to test for this because a disabled item is focusable for accessibility reasons, but it should not do anything when activated
+    await MenuPageObject.clickComponent();
+    await MenuPageObject.waitForMenuToOpen();
+    await MenuPageObject.sendKey(MenuPageObject.getItemSelector(2), Keys.ENTER);
+    await expect(await MenuPageObject.didItemCallbackFire()).toBeFalsy();
+
+    await expect(await MenuPageObject.didAssertPopup()).toBeFalsy(MenuPageObject.ERRORMESSAGE_ASSERT);
+  });
+
+  it('Validate disabled MenuItem onClick() callback does not fire -> Press Space', async () => {
+    // we want to test for this because a disabled item is focusable for accessibility reasons, but it should not do anything when activated
+    await MenuPageObject.clickComponent();
+    await MenuPageObject.waitForMenuToOpen();
+    // fire callback
+    await MenuPageObject.sendKey(MenuPageObject.getItemSelector(2), Keys.SPACE);
+    await expect(await MenuPageObject.didItemCallbackFire()).toBeFalsy();
+
+    await expect(await MenuPageObject.didAssertPopup()).toBeFalsy(MenuPageObject.ERRORMESSAGE_ASSERT);
+  });
+
+  // menuitem navigation with keyboard
+
+  it('Validate MenuItem navigation works correctly -> Press Up + Down', async () => {
+    // open menu
+    await MenuPageObject.clickComponent();
+    await MenuPageObject.waitForMenuToOpen();
+    // send key inputs to items
+    await MenuPageObject.sendKey(MenuPageObject.getItemSelector(1), Keys.DOWN);
+    await expect(await MenuPageObject.itemIsFocused(2)).toBeTruthy();
```

Comment:
Nit: Add a newline after these `expect` statements. Since these are all different actions taking place, it helps the readability to separate-out different UI actions

Reply:
Added new lines for distinct expects with my latest changes.